crates/bitwarden-wasm-internal/src/generators.rs

```diff
@@ -0,0 +1,47 @@
+use std::rc::Rc;
+
+use bitwarden_core::Client;
+use bitwarden_generators::{GeneratorClientsExt, PasswordGeneratorRequest, PassphraseGeneratorRequest};
+use wasm_bindgen::prelude::*;
+
+#[wasm_bindgen]
+pub struct GeneratorClient(Rc<Client>);
+
+impl GeneratorClient {
+    pub fn new(client: Rc<Client>) -> Self {
+        Self(client)
+    }
```

Comment:
⛏️ Documentation is needed here, too. The rule I apply is "if it's scope is any kind of `pub`, it needs documentation".